docker/calculation.go

```diff
@@ -0,0 +1,17 @@
+package docker
+
+import "github.com/docker/docker/api/types"
+
+func calculateMemUsageUnixNoCache(mem types.MemoryStats) float64 {
+	// re implementation of the docker calculation
+	// https://github.com/docker/cli/blob/53f8ed4bec07084db4208f55987a2ea94b7f01d6/cli/command/container/stats_helpers.go#L227-L249
```

Comment:
thanks for including this comment. There won't be any confusion where this came from